src/braket/circuits/angled_gate.py

```diff
@@ -19,25 +19,23 @@
 from functools import singledispatch
 from typing import Optional, Union
 
-from sympy import Float
-
 from braket.circuits.free_parameter_expression import FreeParameterExpression
 from braket.circuits.gate import Gate
 from braket.circuits.parameterizable import Parameterizable
+from sympy import Float
 
 
 class AngledGate(Gate, Parameterizable):
-    """
-    Class `AngledGate` represents a quantum gate that operates on N qubits and an angle.
-    """
+    """Class `AngledGate` represents a quantum gate that operates on N qubits and an angle."""
 
     def __init__(
         self,
         angle: Union[FreeParameterExpression, float],
         qubit_count: Optional[int],
         ascii_symbols: Sequence[str],
     ):
-        """
+        """Inits an `AngledGate`.
```

Comment:
For this and all other `__init__` methods
```suggestion
        """Initializes an `AngledGate`.
```